content/posts/10-year-anniversary.md

```diff
@@ -0,0 +1,65 @@
+image_credit: Photo by <a href="https://unsplash.com/@adigold1?utm_content=creditCopyText&utm_medium=referral&utm_source=unsplash">Adi Goldstein</a> on <a href="https://unsplash.com/photos/selective-focus-photography-of-assorted-color-balloons-Hli3R6LKibo?utm_content=creditCopyText&utm_medium=referral&utm_source=unsplash">Unsplash</a>
+
+Read the Docs was started at a [48-hour open source sprint in 2010](https://www.ericholscher.com/blog/2010/aug/16/announcing-read-docs/), and grew to become a large part of the Python documentation ecosystem within a few years.
+Based on that success and the stress that came with it, we [created a company](https://www.ericholscher.com/blog/2014/oct/24/announcing-read-the-docs-for-business/) to focus on sustainability in 2014.
+We've been working on Read the Docs as a project for 14 years, and a company for 10 years.
+This post will reflect on the last 10 years of trying to make open source sustainable.
+
+## Our sustainability model
+
+We tried a number of different ways to make a sustainable open source project, really a service, over the years.
+The model that we've found that has worked for us currently is a version of the classic open source model used by GitHub, Travis CI, and others:
+
+* We offer a free service (Read the Docs Community) for open source projects, supported by a single non-tracking ad on the documentation.
+* We offer a paid service (Read the Docs Business) for companies that need private repos and additional benefits that make sense for larger organizations.
+
+We tried relying on donations and other optional support, but that didn't work at all, and left us struggling with [mental health issues](https://ericholscher.com/blog/2018/feb/7/the-post-i-never-published/) and burnout.
+We tried a model where we focused on documentation consulting and services, but that mostly took time away from actually working on the core product that open source projects rely on.
+We also [experimented with grant funding](https://blog.readthedocs.com/czi-grant-announcement/),
+which was a much more aligned source of funding allowing work on the core product,
+but is a one-time source of funding.
+
+Our goal is to work on the core product each day,
+and [our current model aligns the incentives](https://ericholscher.com/blog/2016/aug/31/funding-oss-marketing-money/) so that everyone who is using the product is supporting it in some way,
+and we can focus on making it the best it can be for all our users.
+
+## Lessons learned
+
+A few of the most important lessons from the last 10 years:
+
+* You don't get extra points for being open source or bootstrapped. People will compare you to proprietary, venture-backed services that have raised many millions of dollars. **We compete by focusing on documentation, a niche too small for most venture-backed businesses**. Venture-backed companies have to keep adding unrelated features in order to grow into other areas, and we can focus on our core competency.
```

Comment:
I'd say we actually do get _a fair amount_ of points for being open source at least. There is a lot of good will there because we are open source and support open source, and customers quite often come over from community. But this still makes the most sense framed around just bootstrapping too.

 ```suggestion
* You don't get extra points for being bootstrapped. People will compare you to proprietary, venture-backed services that have raised many millions of dollars. **We compete by focusing on documentation, a niche too small for most venture-backed businesses**. Venture-backed companies have to keep adding unrelated features in order to grow into other areas, and we can focus on our core competency.
```

There is something to say about open source though.